polaris/evaluate/_metric.py

```diff
@@ -30,6 +31,30 @@ def spearman(y_true: np.ndarray, y_pred: np.ndarray):
     return stats.spearmanr(y_true, y_pred).statistic
 
 
+def absolute_average_fold_error(y_true: np.ndarray, y_pred: np.ndarray):
```

Comment:
This is actually needed, because the docs will fail otherwise. 

```suggestion
def absolute_average_fold_error(y_true: np.ndarray, y_pred: np.ndarray) -> float:
```